packages/core/src/controllers/relayer.ts

```diff
@@ -131,9 +131,20 @@ export class Relayer extends IRelayer {
     this.events.removeListener(event, listener);
   }
 
+  public async transportClose() {
+    this.transportExplicitlyClosed = true;
+    await this.provider.connection.close();
+  }
+
+  public async transportOpen(relayUrl: string = RELAYER_DEFAULT_RELAY_URL) {
```

Comment:
I think the `relayUrl` param shouldn’t default to our constant here but should be undefined by default, because of this scenario:

1. Create an initial client with custom relayUrlA
2. Close transport
3. Reopen but don’t provide a relayUrl
4. -> transportOpen quietly overwrites our relayUrlA value with the default relayUrl

Reply:
Good catch, thank you!